django-backend/fecfiler/shared/test_transactions.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
```

Comment:
I would leave this out for now until we have a clear path forward for dividing up the schedules on the api